PVC tests failing "concurrent.futures._base.TimeoutError"

The tests in below files failed with `concurrent.futures._base.TimeoutError` while trying to get fio results
 - tests.manage.pv_services.test_raw_block_pv.py
 - tests/manage/pv_services/test_run_io_multiple_dc_pods.py
 - tests/manage/pv_services/test_run_io_multiple_pods.py

Failures were observed in RUN ID: 1691418879, 1691416825 and 1691416926

Error message from two of the failures:
```
>       pods, pvcs, pvs = self.raw_block_pv

tests/manage/pv_services/test_raw_block_pv.py:152: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/manage/pv_services/test_raw_block_pv.py:145: in raw_block_pv
    get_fio_rw_iops(pod)
ocs_ci/ocs/resources/pod.py:1265: in get_fio_rw_iops
    fio_result = pod_obj.get_fio_results(120)
ocs_ci/ocs/resources/pod.py:144: in get_fio_results
    result = self.fio_thread.result(timeout)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = None, timeout = 120

    def result(self, timeout=None):
        """Return the result of the call that the future represents.
    
        Args:
            timeout: The number of seconds to wait for the result if the future
                isn't done. If None, then there is no limit on the wait time.
    
        Returns:
            The result of the call that the future represents.
    
        Raises:
            CancelledError: If the future was cancelled.
            TimeoutError: If the future didn't finish executing before the given
                timeout.
            Exception: If the call raised then that exception will be raised.
        """
        try:
            with self._condition:
                if self._state in [CANCELLED, CANCELLED_AND_NOTIFIED]:
                    raise CancelledError()
                elif self._state == FINISHED:
                    return self.__get_result()
    
                self._condition.wait(timeout)
    
                if self._state in [CANCELLED, CANCELLED_AND_NOTIFIED]:
                    raise CancelledError()
                elif self._state == FINISHED:
                    return self.__get_result()
                else:
>                   raise TimeoutError()
E                   concurrent.futures._base.TimeoutError

/usr/lib64/python3.8/concurrent/futures/_base.py:446: TimeoutError
```

```
>           get_fio_rw_iops(dc_pod)

/home/jenkins/workspace/qe-deploy-ocs-cluster-prod/ocs-ci/tests/manage/pv_services/test_run_io_multiple_dc_pods.py:71: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/home/jenkins/workspace/qe-deploy-ocs-cluster-prod/ocs-ci/ocs_ci/ocs/resources/pod.py:1265: in get_fio_rw_iops
    fio_result = pod_obj.get_fio_results(120)
/home/jenkins/workspace/qe-deploy-ocs-cluster-prod/ocs-ci/ocs_ci/ocs/resources/pod.py:144: in get_fio_results
    result = self.fio_thread.result(timeout)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = None, timeout = 120

    def result(self, timeout=None):
        """Return the result of the call that the future represents.
    
        Args:
            timeout: The number of seconds to wait for the result if the future
                isn't done. If None, then there is no limit on the wait time.
    
        Returns:
            The result of the call that the future represents.
    
        Raises:
            CancelledError: If the future was cancelled.
            TimeoutError: If the future didn't finish executing before the given
                timeout.
            Exception: If the call raised then that exception will be raised.
        """
        try:
            with self._condition:
                if self._state in [CANCELLED, CANCELLED_AND_NOTIFIED]:
                    raise CancelledError()
                elif self._state == FINISHED:
                    return self.__get_result()
    
                self._condition.wait(timeout)
    
                if self._state in [CANCELLED, CANCELLED_AND_NOTIFIED]:
                    raise CancelledError()
                elif self._state == FINISHED:
                    return self.__get_result()
                else:
>                   raise TimeoutError()
E                   concurrent.futures._base.TimeoutError

/usr/lib64/python3.8/concurrent/futures/_base.py:446: TimeoutError
```

--- a/ocs_ci/ocs/resources/pod.py
+++ b/ocs_ci/ocs/resources/pod.py
@@ -1316,7 +1316,7 @@ def get_fio_rw_iops(pod_obj):
     Args:
         pod_obj (Pod): The object of the pod
     """
-    fio_result = pod_obj.get_fio_results(120)
+    fio_result = pod_obj.get_fio_results()
     logger.info(f"FIO output: {fio_result}")
     logger.info("IOPs after FIO:")
     logger.info(f"Read: {fio_result.get('jobs')[0].get('read').get('iops')}")

